packages/browser/src/login/oidc/redirectHandler/AuthCodeRedirectHandler.ts

```diff
@@ -69,6 +69,48 @@ export async function exchangeDpopToken(
   });
 }
 
+// We
+const DEFAULT_LIFESPAN = 1800;
+
+/**
+ * Stores the resoure server session information in local storage, so that they
+ * can be leveraged on refresh.
```

Comment:
The word 'leverage' always sets off alarm bells in my head (ever since I read "Why Business People Speak Like Idiots"!):
```suggestion
 * can be used on refresh.
```

Reply:
f90ee08859fb72fb6288d3976349128f38aa1673